### ISSUE TITLE
Cursor blinking in hexview, ignoring desktop setting

### ISSUE THREAD

**Issue by kermitfrog** [solemnwarning/rehex#112]
I'm using KDE on linux and have turned off cursor blinking is everywhere I can.
Despite this, the cursor in the rehex hex view, is blinking. (it does not e.g. in the save as input field)
As far as I know, the corresponding setting, that should be used by wxWidgets (and is set by KDE) is in 
~/.config/gtk-3.0/settings.ini
The line:
gtk-cursor-blink=0

**Comment by solemnwarning** [comment]
This isn't Linux specific; it never occured to me the OS would have cursor blink settings I should query when I wrote the custom controls.

**Comment by solemnwarning** [comment]
Putting this here as a reminder for myself so I don't have to go digging through the Git history to find it next time:

```
Turns out wxCaret::GetBlinkTime() isn't very reliable - on Windows it
doesn't properly handle the cursor blink being disabled and on GTK/Mac
it has a hardcoded default of 500ms.

Calling wxSystemSettings::GetMetric() with wxSYS_CARET_ON_MSEC and
wxSYS_CARET_OFF_MSEC looks to be the right way forward, but it isn't
implemented before wxWidgets 3.1.1.
```

I'll probably fix this after wxWidgets 3.2 comes out (ETA Sep 2021), with a fallback for older GTK versions on Linux.

**Comment by solemnwarning** [comment]
Merged in a28b74a6c03719dfb407332ba64f445ba91923a6
